packages/web-components/src/button/button.options.ts

```diff
@@ -0,0 +1,53 @@
+import { ButtonOptions, ValuesOf } from '@microsoft/fast-foundation';
+
+/**
+ * ButtonAppearance constants
+ * @public
+ */
+export const ButtonAppearance = {
```

Comment:
I noticed the tokens I looked at still use `neutral` and `stealth` I'm sure there will be parity, but we should make sure the tokens align with the usage

Reply:
The tokens don't necessarily map directly to appearances from what I can tell - here are the latest types I see in code: https://github.com/microsoft/fluentui/blob/1a45088ee3492f352e23748539bf207af57f99a8/packages/react-components/react-button/src/components/Button/Button.types.ts#L32